nearlib/account.js

```diff
@@ -151,5 +151,40 @@ class Account {
     async viewAccount (accountId) {
         return await this.nearClient.viewAccount(accountId);
     }
+
+    /**
+     * Returns full details for a given account.
+     * @param {string} accountId id of the account to look up 
+     */
+    async getAccountDetails(accountId) {
+        accountId; // make linter happy
+        // Hardcoded for now
+        const result = {
+            authorizedApps : [
+                {
+                    contractId: 'cryptomonsters',
+                    name: 'Monsters',
+                    balance: 10,
+                },
+                {
+                    contractId: 'pay_mo',
+                    name: 'Paymo',
+                    balance: 20 
+                }
+            ],
+            transactions : [
+                {
+                    id: '24359uy1dfsgwr',
+                    transactionType: 'TRANSFER_TOKENS', // to be converted to a description string on front end for easier localization
```

Comment:
let's just call this `type`, as e.g. we don't call `id` field `transactionId`

Reply:
done